dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCAttestationType.scala

```diff
@@ -0,0 +1,29 @@
+package org.bitcoins.dlc.oracle
+
+import org.bitcoins.crypto.CryptoUtil
+import scodec.bits.ByteVector
+
+sealed trait DLCAttestationType {
+  def bytes: ByteVector
+}
+
+case class EnumAttestation(outcome: String) extends DLCAttestationType {
+  def bytes: ByteVector = CryptoUtil.serializeForHash(outcome)
+}
+
+case class RangeAttestation(outcome: Long) extends DLCAttestationType {
+  def bytes: ByteVector = CryptoUtil.serializeForHash(outcome.toString)
+}
+
+case class LargeRangeSignAttestation(positive: Boolean)
```

Comment:
I don't really understand what the purpose of this attestation is? My understanding with all other classes in this file is they are actual attestations themselves. 

For number attestsations, do we imagine there would be a `Vector[RangeAttestation]` that commits to each individual digital and that is what is published? 

So in reality, we would need to publish something like `(LargeRangeSignAttestation, Vector[LargeRangeAttestation])`?

Reply:
This just an internal type for passing around data that we will sign. When we publish the actual attestation it will be a `Vector[SchnorrDigitalSignature]`, this type just holds the preimage to the hash of what we will be signing.

I think the confusing comes from the name of this type. Idk what would be better tho